### ISSUE TITLE
Accessibility: scrollable regions should be keyboard-focusable

### ISSUE THREAD

**Issue by isaacmattern** [atomiks/rehype-pretty-code#108]
While running axe dev tools on code blocks generated by rehype-pretty-code (on my own site and on rehype-pretty-code's documentation site), I'm given the error "Scrollable region must have keyboard access".

I believe that this issue can be fixed by adding an attribute of tabindex="0" to each `<pre>` element which is horizontally scrollable. 

[Example of problem and fix by adding tabindex attribute](https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable/)

[Axe Devtools](https://www.deque.com/axe/devtools/)

<img width="907" alt="Screenshot 2023-09-19 at 11 19 37 PM" src="https://github.com/atomiks/rehype-pretty-code/assets/77303978/99acd497-aa52-4083-9d75-ec4a30dcbfc9">
